internal/lock_test.go

```diff
@@ -33,7 +33,6 @@ func TestNewLockValid(t *testing.T) {
 	assert.Equal(t, "sha256-asdasdasd", statement.Integrity)
 	assert.Equal(t, []string{"tag1", "tag2"}, statement.Tags)
 }
-
```

Comment:
Why the line removal?

Reply:
I removed it accidentally. I’ll redo that, sorry!